From: SeongJae Park <sj@kernel.org>
Date: Mon, 24 Feb 2025 12:12:22 -0800
Subject: [PATCH] mm/damon/core: fix wrong closing brace location

This can make damon_nr_regions selftest hangs, since it causes
after_aggregation() never invoked.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 mm/damon/core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mm/damon/core.c b/mm/damon/core.c
index db9067b69c10..168e8bc9e146 100644
--- a/mm/damon/core.c
+++ b/mm/damon/core.c
@@ -2427,10 +2427,10 @@ static int kdamond_fn(void *data)
 					ctx->attrs.aggr_samples *
 					ctx->attrs.intervals_goal.aggrs;
 				kdamond_tune_intervals(ctx);
+			}
 			if (ctx->callback.after_aggregation &&
 					ctx->callback.after_aggregation(ctx))
 				break;
-			}
 		}
 
 		/*
-- 
2.39.5

